open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb

```diff
@@ -0,0 +1,2329 @@
+{
```

Comment:
<p>Use a 1-point font for the title</p><p>#Indexing blablabla</p><p> </p><p>The following headings have to be changed </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>